package.json

```diff
@@ -89,6 +89,7 @@
     "protractor": "^7.0.0",
     "protractor-jasmine2-screenshot-reporter": "^0.5.0",
     "puppeteer": "^7.0.0",
+    "randomstring": "^1.1.5",
```

Comment:
Is it possible to do this without adding a dependency? This would probably be pretty simple to write ourselves even. I think @vojtechjelinek manages our dependencies, so I'll defer to him on whether we should avoid adding this dependency

Reply:
Yep, I would agree to create this ourselves. We don't want to get ourselves into a [left-pad kind of incident](https://arstechnica.com/information-technology/2016/03/rage-quit-coder-unpublished-17-lines-of-javascript-and-broke-the-internet/).

Reply:
Alright, `Math.random().toString(36).substring(2,8)` is the solution I've implemented.